src/main/java/jalau/at18/kingoftokyo/DiceFaceSameCounter.java

```diff
@@ -0,0 +1,19 @@
+package jalau.at18.kingoftokyo;
+
+public class DiceFaceSameCounter {
+    private DiceHand diceHand;
+
+    public DiceFaceSameCounter(DiceHand listDiceHand) { //lista del random
+        this.diceHand = listDiceHand;
+    }
+
+    public DiceIndentifier diceFaceCounter() { //agarra la lista del ramdom y cuenta la cantidad caras del dado repetidos
```

Comment:
You can keep the comment, but please change it to english. As alternative, you can just remove it